components/shapediver/appbuilder/widgets/AppBuilderAccordionUiWidgetComponent.tsx

```diff
@@ -0,0 +1,51 @@
+import AppBuilderWidgetsComponent from "@AppBuilderShared/components/shapediver/appbuilder/widgets/AppBuilderWidgetsComponent";
+import {IAppBuilderWidgetPropsAccordionUi} from "@AppBuilderShared/types/shapediver/appbuilder";
+import {Accordion} from "@mantine/core";
+import React, {useState} from "react";
+
+type Props = IAppBuilderWidgetPropsAccordionUi & {
+	namespace: string;
+};
+
+export default function AppBuilderAccordionUiWidgetComponent({
+	namespace,
+	items,
+	multiple,
+	defaultValue,
+	value,
+}: Props) {
+	// Local state for uncontrolled mode
+	const [localValue, setLocalValue] = useState<string | string[] | null>(
+		defaultValue || null,
+	);
+
+	// Determine current value and setValue function based on mode
+	const currentValue = value !== undefined ? value : localValue;
+
+	const handleValueChange = (newValue: string | string[] | null) => {
+		// Update local state (uncontrolled mode)
+		setLocalValue(newValue);
+	};
```

Comment:
@LemToUp I think there is a problem here, because if `value` is defined, `currentValue` will always be set to `value` and therefore the user can't change the accordion state. This is how it's meant to work: 

  * When the component is newly instantiated: If `value` or `defaultValue` are defined, the state of the accordion is initialized using them (value takes precedence)
  * After a computation (which might change the given props): 
    * If `defaultValue` changes, the state of the accordion does not change
    * If `value` changes, the state of the accordion is updated accordingly

Reply:
@snabela you are right. Thank you.
https://github.com/shapediver/AppBuilderShared/pull/156/commits/466444399f48ce994b862d9a2a9ceaf0bc8f07f7